doc/user/methods/random_forest.md

```diff
@@ -367,6 +373,41 @@ std::cout << "Probabilities of each class: " << probabilities.t();
 
 ---
 
+Train a `RandomForest` with the `SequentialBootstrap` strategy.
+
+```c++
+// 1000 random points in 10 dimensions.
+arma::mat dataset(10 /* rows */, 1000 /* cols */, arma::fill::randu);
+// Random labels for each point, totaling 5 classes.
+arma::Row<size_t> labels =
+arma::randi<arma::Row<size_t>>(1000, arma::distr_param(0, 4));
+
+arma::umat intervals(labels.n_cols, 2);
+for (arma::uword i(0); i < labels.n_cols; ++i) {
+  // Create intervals of random length.
+  intervals(i, 0) = i;
+  intervals(i, 1) = i + std::rand();
+  if (intervals(i, 1) > 1000) {
+    intervals(i, 1) = 1000;
+  }
+}
```

Comment:
Can you make this (and other code) match the mlpack style guide?

Also I know it maybe makes the example a little less boring, but I think it would be clearer if you just hardcoded two or three intervals and then made the `SequentialBootstrap` object.  For the setup, too, maybe it makes sense to add a line or two in the comments like "this is random data, but in a real-world scenario this might be financial time series data that consists of a number of events...", this type of thing to give a bit of extra clarity on the context in which this might be used.